bats_test/rename/rename_colliding_name.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Depends on ./rename_file.sh
+
+rename_colliding_name() {
+    # File ID at Drive with ID: c0c8ba1c-efc5-420d-a07c-a755dc67f6b2
+    PUBLIC_FILE_ID="290a3f9a-37b2-4f0f-a899-6fac983833b3"
```

Comment:
I think this would fail with any wallet that didn't upload that particular file id because of the owner assertion

Reply:
We might need a $PUB_FILE_ID variable being set here from the docker wallet setup script

Reply:
Working on this